app/models/ucas_match.rb

```diff
@@ -10,14 +10,15 @@ class UCASMatch < ApplicationRecord
     resolved_on_apply: 'resolved_on_apply',
     resolved_on_ucas: 'resolved_on_ucas',
     manually_resolved: 'manually_resolved',
+    no_application_choice: 'no_application_choice',
```

Comment:
I think we might need to surface this new value in the UCAS views and action list

Reply:
Added a 'bad data' section to the dashboard. Do you think we need a new label?

Reply:
'No application choice' UCAS matches already show up, can be filtered etc.